Create entitlement-submission doc (#5754)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/update-oin-app/main/index.md

* [Universal Logout](https://developer.okta.com/docs/guides/oin-universal-logout-overview/)

* [Entitlement Management](https://developer.okta.com/docs/guides/scim-with-entitlements/main/)

> > **Note:** You can also update OIDC, SAML 2.0, and SCIM 2.0 integrations with the [OIN Wizard](/docs/guides/update-oin-app/) that were originally submitted through the [OIN Manager](/docs/guides/submit-app/).

Reviewer: ```suggestion
> **Note:** You can use the [OIN Wizard](/docs/guides/update-oin-app/) to update OIDC, SAML 2.0, and SCIM 2.0 integrations that were originally submitted through the [OIN Manager](/docs/guides/submit-app/).
```
